posawesome/posawesome/api/purchase_orders.py

```
@@ -303,73 +303,98 @@ def get_supplier_info(supplier):
+            pii.uom,
+            pi.currency,
+            pi.name AS invoice,
+            pi.posting_date,
+            pi.supplier
+        FROM `tabPurchase Invoice Item` pii
+        JOIN `tabPurchase Invoice` pi ON pi.name = pii.parent
+        WHERE pi.docstatus = 1
+          AND pii.item_code IN %s
+          {supplier_clause}
+        ORDER BY pi.posting_date DESC, pi.creation DESC
+        """,
```

Ruff (0.15.10)

[error] Possible SQL injection vector through string-based query construction

(S608)